Complete Rewriting of wiilink.md (#400)
RiiConnect24/Wii-Guide: _pages/en_US/wiilink.md

{% include toc title="Table of Contents" %}

> For support on WiiLink24, please visit the [WiiLink24 Discord Server](https://discord.com/invite/reqUMqxu8D). {: .notice--info}

Reviewer: ```suggestion
For support on WiiLink24, please visit the [WiiLink24 Discord Server](https://discord.com/invite/reqUMqxu8D). 
{: .notice--info}
```

4. Start up the Homebrew Channel and enter yawmME (YAWM ModMii Edition).
> Select your SD Card or USB Drive accordingly within the Homebrew Channel.

5. Highlight all of the WADs with `WiiLink` in the name and press + to select them. When all of them are selected, press A twice to install the WADs.
> If you have previously installed the RiiConnect24 versions of the WiiConnect24 Channels, installing WiiLink's versions will overwrite them. If they have not been overwritten, try uninstalling the RiiConnect24 versions, and reinstalling them with the WiiLink24 versions.

> If you are installing the Food Channel or Photo Prints Channel, you may also have a file called `WiiLink SPD`. This file will have either have `(Wii)` or `(vWii)` depending on which console youre installing it on. Please install the correct version of the SPD onto your console.

6. After it is successfully installed, press the HOME Button to exit back to the Homebrew Channel. Press the HOME Button again and select `"Exit to System Menu"`.

> ### Congratulations! You've installed WiiLink24 onto your console! Enjoy what WiiLink has to offer to your hearts content.

Reviewer: ```suggestion
Congratulations! You've installed WiiLink24 onto your console! Enjoy what WiiLink has to offer to your hearts content.
{: .notice--success}
```

> > For more support on our service, please visit the [WiiLink24 Discord Server](https://discord.com/invite/reqUMqxu8D).

Reviewer: ```suggestion
For more support on our service, please visit the [WiiLink24 Discord Server](https://discord.com/invite/reqUMqxu8D).
{: .notice--info}
```

Want to order Domino's Pizza with WiiLink's Food Channel? [Follow this guide!](wiilink-demae-dominos)
{: .notice--info}

For PAL (European) Wii's<br>

> In Wii Settings, please set your TV Resolution to 60 HZ (480p/480i). You can find this in `Screen -> TV Resolution`. Failure to do this will cause a pink bar will appear at the bottom of the screen, obscuring vision.

Reviewer: ```suggestion
In Wii Settings, please set your TV Resolution to 60 HZ (480p/480i). You can find this in `Screen -> TV Resolution`. Otherwise, a pink bar will display at the bottom of the screen, obscuring vision.
```

Author: Suggestion has been added to the fix👍

4. Start up the Homebrew Channel and enter yawmME (YAWM ModMii Edition).
> Select your SD Card or USB Drive accordingly within the Homebrew Channel.

5. Highlight all of the WADs with `WiiLink` in the name and press + to select them. When all of them are selected, press A twice to install the WADs.
> If you have previously installed the RiiConnect24 versions of the WiiConnect24 Channels, installing WiiLink's versions will overwrite them. If they have not been overwritten, try uninstalling the RiiConnect24 versions, and reinstalling them with the WiiLink24 versions.

> If you are installing the Food Channel or Photo Prints Channel, you may also have a file called `WiiLink SPD`. This file will have either have `(Wii)` or `(vWii)` depending on which console youre installing it on. Please install the correct version of the SPD onto your console.

6. After it is successfully installed, press the HOME Button to exit back to the Homebrew Channel. Press the HOME Button again and select `"Exit to System Menu"`.

> ### Congratulations! You've installed WiiLink24 onto your console! Enjoy what WiiLink has to offer to your hearts content.

Reviewer: ```suggestion
Congratulations! You've installed WiiLink24 onto your console! Enjoy what WiiLink has to offer to your hearts content.
```
